File: practice-2017-10-06-android/CleanArchitectureAndTDDDemo/core/src/main/java/com/example/entities/Cat.java
package com.example.entities;

import java.util.UUID;

/**
 * Created by MyndDev on 06/10/2017.
 */

public class Cat {
    private UUID id;
    private String name;
    private String title;

    public Cat(UUID id) {
        this.id = id;
    }

    public UUID getId() {
        return id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }
}

File: 2018-01-PSITS-SultanKudarat/practice-2018-01-13/Decoupled_Final/CatsProject.Domain/Entities/Cat.cs
﻿using System;

namespace Domain.Entities
{
    public class Cat
    {
        public Cat(Guid id)
        {
            Id = id;
        }

        public Guid Id { get; private set; }
        public string Name { get; set; }
        public string Title { get; set; }
    }
}